tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -3396,13 +3396,13 @@ protected function createPaymentInstrument($params = array(), $financialAccountN
       ),
       $params
     );
-    $newPaymentInstrument = $this->callAPISuccess('OptionValue', 'create', $params);
+    $newPaymentInstrument = $this->callAPISuccess('OptionValue', 'create', $params)['id'];
 
     $relationTypeID = key(CRM_Core_PseudoConstant::accountOptionValues('account_relationship', NULL, " AND v.name LIKE 'Asset Account is' "));
 
     $financialAccountParams = [
       'entity_table' => 'civicrm_option_value',
-      'entity_id' => key($newPaymentInstrument),
+      'entity_id' => $newPaymentInstrument,
```

Comment:
Or directly call here:
```
'entity_id' => $this->callAPISuccess('OptionValue', 'create', $params)['id'],
```

Reply:
please include this change in a separate sub-/PR.

Reply:
It wouldn't pass locally for me without fixing this - depending how I ran it

Reply:
Oh ok